code/CSS assignment-Grid/style.css

```diff
@@ -0,0 +1,41 @@
+#wrapper {
+    font-size: 2em;
+    gap: 1rem;
+    display: grid;
+    grid-template-columns: 10rem auto 10rem;
+    grid-template-rows: 20vh 90vh 20vh;
+    grid-template-areas:
+    "header header header"
+    "sidebar-left main sidebar-right"
+    "footer footer footer";
+}
+
+#header{
+    grid-area: header;
+    background-color:#ddd
```

Comment:
It wasn't necessary to restate the background color, all of these items already have that property